### ISSUE TITLE
Please add a row what happens if user has mailbox is without hybrid

### ISSUE THREAD

**Issue by Larryngolog** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#3403]
We are missing documentation what is the experience if user does not have Exchange Online mailbox nor there is a hybrid with Exchange on-prem.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8456689d-6f04-fa5e-4f51-f413cb0b9d89
* Version Independent ID: e9b85126-0db7-4378-ad44-514e52f9fde1
* Content: [How Exchange and Microsoft Teams interact - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/exchange-teams-interact)
* Content Source: [Teams/Exchange-Teams-interact.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/Exchange-Teams-interact.md)
* Service: **msteams**
* GitHub Login: @dstrome
* Microsoft Alias: **dstrome**

**Comment by scanum** [comment]
@Larryngolog Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by dariomws306** [comment]
Hi @Larryngolog, thank you for your feedback.

I think the information is in the article. Synchronization with Azure Active Directory Connect is a requirement for the last row but not Exchange Hybrid deployment.

**Comment by Larryngolog** [comment]
@dariomws306 unfortunately this is not the case. According to documentation below we also need to configure OAuth, but some of my customers claim that only after configuring hybrid we will be able to see the mailbox.
Apart from that let me rephrase my remark - we need information what will not work if we do not have the ability to connect to local Exchange - for example we are on earlier versions of Exchange, no identity sync or no Exchange at all. A lot of organizations have still mail on-prem but want to try Teams. We need to know what will not work if we do not have mailbox online AND we cannot connect for various reasons to local Exchange server.

**Comment by dariomws306** [comment]
@Larryngolog Than you for your comment.

Yes, Azure AD Connect and OAuth are both a requirement, it's true. If after that there are some customers that still don't have interaction between Teams and Exchange on-premises, they/you should open a support case as here we can help with document improvements, not with specific issues.

About the second paragraph, I understand that if there isn't a hybrid nor AD sync, you won't have integration between Teams and Exchange on-premises. You can have some integration between clients (Outlook and Teams) but not server/service integration.
For example if we should a row for this scenario, it should be "no" for each column.

**Comment by scanum** [comment]
@Larryngolog Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.
